dbschema/src/main/java/org/folio/rest/persist/ddlgen/Index.java

```diff
@@ -129,7 +134,12 @@ private static  boolean appendExpandedSimpleTerm(String[] rawExpandedTerm, Strin
   private static boolean appendExpandedArrayTerm(String[] rawExpandedTerm, StringBuilder expandedTerm, int currentTermIdx,
       int tokenIdx) {
     String arrowToken = "->";
-    expandedTerm.append(arrowToken).append("'").append(rawExpandedTerm[currentTermIdx].substring(0,tokenIdx)).append("',").append("'").append(rawExpandedTerm[currentTermIdx+1]).append("'");
+    if(currentTermIdx == rawExpandedTerm.length - 1) {
+      arrowToken = "->";
+      expandedTerm.append(arrowToken).append("'").append(rawExpandedTerm[currentTermIdx].substring(0,tokenIdx)).append("'");
+    } else {
+      expandedTerm.append(arrowToken).append("'").append(rawExpandedTerm[currentTermIdx].substring(0,tokenIdx)).append("',").append("'").append(rawExpandedTerm[currentTermIdx+1]).append("'");
+    }
```

Comment:
appendExpandedTerm, formatExpandedTerm, appendExpandedSimpleTerm, appendExpandedArrayTerm are complicated code and difficult to understand. They reimplement the code in index2sqlText and index2sqlJson from Cql2PgJson. My suggestion is to wait until "RMB-518: Extract index2sqlText and index2sqlJson for reuse" #563 has been merged. That will make the implementation in Index.java very easy.

Reply:
Please do not make term manipulation in dbschema.. That belongs to cql2pgjson ... Is it difficult to make that switch?

Reply:
Most guidelines have a blank after a reserved word.. Eg "if" in this case.

Reply:
I have re worked the code to us the utility functions